clients/src/main/java/org/apache/kafka/clients/consumer/ConsumerConfig.java

```diff
@@ -316,6 +320,11 @@
                                         atLeast(1),
                                         Importance.MEDIUM,
                                         MAX_POLL_RECORDS_DOC)
+                                .define(EXCLUDE_INTERNAL_TOPICS_CONFIG,
+                                        Type.BOOLEAN,
+                                        true,
```

Comment:
@hachikuji I have a comment higher up in this PR that is relevant. The old behavior (old consumer) was to exclude them by default. The thought was this was likely the most common use case for users and it may be worth changing now as many users likely just haven't run into the issue yet.